setup.py

```diff
@@ -41,8 +42,8 @@
 )
 
 install_requires = [
-    'aiohttp>3.6.2,<3.9.0',
-    'requests>=2.24.0,<3.0.0',
+    'aiohttp~=3.6',
```

Comment:
it is recommended to upgrade to at least version 3.10.5 or higher to ensure compatibility with Python 3.12